bootstrap-salt.sh

```diff
@@ -18,7 +18,7 @@
 #======================================================================================================================
 set -o nounset                              # Treat unset variables as an error
 
-__ScriptVersion="2017.05.24"
+__ScriptVersion="2017.07.25"
```

Comment:
Can you remove this change? We only update this release number when we create a new stable release for the bootstrap script.